parquet-hadoop/src/main/java/org/apache/parquet/filter2/statisticslevel/StatisticsFilter.java

```diff
@@ -68,8 +68,8 @@
   private static final boolean BLOCK_CANNOT_MATCH = true;
 
   public static boolean canDrop(FilterPredicate pred, List<ColumnChunkMetaData> columns) {
-    checkNotNull(pred, "pred");
-    checkNotNull(columns, "columns");
+    checkNotNull(pred, "predicate cannot be null");
+    checkNotNull(columns, "columns cannot be null");
```

Comment:
the second parameter is just the name of the parameter.
https://github.com/apache/parquet-mr/blob/6dad1e3bd0e277f5b5e5e2a0720f474271c1648d/parquet-common/src/main/java/org/apache/parquet/Preconditions.java#L38
if renaming pred, you should rename in in the method signature as well.